wpeverest/everest-forms
# Feature - Oxygen builder integration

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces the oxygen builder integration module.
Note :- Oxygen builder zip in the jira link and install it.

### How to test the changes in this Pull Request:

1. Goto dashboard > feature > oxygen > enable it.
2. Edit a page with oxygen builder > everest forms widget > drag froms
3. Choose a form and apply some style from controller.
4. Verify in editor as well as in frontend after previewing the page.

### Types of changes:


### Other information:


### Changelog entry

>Feature - Oxygen builder integration.
